### PR TITLE
Add SequenceNumber and MessageId to events

### PR DESCRIPTION
**Pull request by miguelruizdev**
As stated here: https://github.com/Activiti/Activiti/issues/2142
Depending on this other PR: https://github.com/Activiti/activiti-cloud-api/pull/59

### CODE REVIEW

**Comment by miguelruizdev** [comment]
Right now, there's coverage for task events.
If I get the green light on this, I will start with the rest of events, entities and converters.

File: activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/streams/AuditConsumerChannelHandlerImplTest.java [hunk 1]
```diff
@@ -18,15 +18,29 @@
...
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.mockito.Spy;
+import org.springframework.messaging.handler.annotation.Headers;
 
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (salaboy) [inline activiti-cloud-services-audit-jpa/src/test/java/org/activiti/cloud/services/audit/jpa/streams/AuditConsumerChannelHandlerImplTest.java]: @miguelruizdev ok .. here we need an extra test where the HashMap is not empty and contains some values
